ConnectionManager.lib.nut

```diff
@@ -340,7 +355,7 @@ class ConnectionManager {
             imp.wakeup(0, connect.bindenv(this));
         } else {
             // Brutally stop trying to connect
-            server.disconnect();
+            // server.disconnect(); // TODO: Disable it for now (under test)
```

Comment:
I tried commenting this out when testing Lunera app, and it did not work as I expected. Please test.

Reply:
Didn't mean to leave it. Uncommented back. Thanks!